dio/test/browser_test.dart

```diff
@@ -0,0 +1,20 @@
+@TestOn('chrome')
+import 'dart:typed_data';
+
+import 'package:dio/browser.dart';
+import 'package:dio/dio.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('with credentials', () async {
+    final browserAdapter = BrowserHttpClientAdapter();
+    browserAdapter.withCredentials = true;
+    final opts = RequestOptions();
+    final testStream =
+        Stream<Uint8List>.periodic(Duration(seconds: 1), (x) => Uint8List(x));
+    final cancelFuture = opts.cancelToken?.whenCancel;
+
+    browserAdapter.fetch(opts, testStream, cancelFuture);
+    expect(browserAdapter.xhrs.first.withCredentials, isTrue);
```

Comment:
Should be `every`?

Reply:
They are equivilant because in the test case there is only one element. But surely, from the logic of programming, 'every' is more appropriate.